system/lib/libc/musl/arch/emscripten/syscall_arch.h

```diff
@@ -138,3 +138,4 @@ long __syscall333(int which, ...);
 long __syscall334(int which, ...);
 long __syscall340(int which, ...);
 
+#undef SYS_futimesat
```

Comment:
Not fully sure.
Seems like because the implementation of `utimensat`, which we actually use, changed to include this reference. The other syscalls we don't use presumably get stripped by DCE before causing compile errors.
I still think undefining `SYS_futimesat` here is reasonable, because it eliminates a new codepath in `utimensat` that we don't support yet.